lib/[celeb]/Md/InterestingProfiles/InterestingProfiles.tsx

```diff
@@ -1,21 +1,19 @@
-import { Container } from '@mui/material';
 import React from 'react';
 import { CelebGallery } from '~/lib/components/CelebGallery';
 import { Separator } from '~/lib/components/Separator';
 import { useCelebContext } from '~/lib/components/StaticPropsContextProvider';
-import s from './styles.module.scss';
 
 export const InterestingProfiles = () => {
   const context = useCelebContext();
   const relatedPeople = context.celeb.oldContent!.relatedPeople;
 
   return (
-    <div className={s.InterestingProfiles}>
+    <div className="px-4 lg:px-0 max-w-home-container mx-auto">
```

Comment:
why do we need to remove the x-padding for large screens?

Reply:
I think we only need this padding on mobile view because its taking some space in large screen that is not necessary in my opinion.<br><br>

Large screen view: 
![image](https://user-images.githubusercontent.com/68635840/165442380-f3fbf8b5-4b81-4ac5-b79b-e1c26c24a0d5.png)

Mobile screen view:
![image](https://user-images.githubusercontent.com/68635840/165442508-86a5542c-6014-4f1d-b2f1-7ddf5d826c0b.png)